tests/integration/integration.go

```diff
@@ -26,7 +26,6 @@ import (
 	_ "github.com/dapr/dapr/tests/integration/suite/actors"
 	_ "github.com/dapr/dapr/tests/integration/suite/daprd"
 	_ "github.com/dapr/dapr/tests/integration/suite/healthz"
-	_ "github.com/dapr/dapr/tests/integration/suite/metadata"
```

Comment:
This daprd metadata test was in the wrong location `suite/metadata` -> `suite/daprd/metadata`.